Authentication_and_Authorization/hello-security/db-init.sql

```diff
@@ -1,8 +1,23 @@
 CREATE DATABASE IF NOT EXISTS `hello-security`;
-
 use `hello-security`;
 
 create table Person (email varchar(255), firstname varchar(255), password varchar(255), role varchar(255));
 
 INSERT INTO person (email, firstname, password, role) values ('asdf@asd', 'kir4o', 'pass', 'student');
```

Comment:
Currently the credentials provided do not work because of the hashing. You may want to adapt them. Example reference: https://github.com/tancheto/WEB2020-php/pull/9/files